RELEASE/scripts/autoscend.ash

```diff
@@ -353,18 +353,28 @@ boolean LX_burnDelay()
 		if(wannaVote)
 		{
 			auto_log_info("Burn some delay somewhere (voting), if we found a place!", "green");
+			set_property("auto_nextEncounter",get_property("_voteMonster"));
 			if(auto_voteMonster(true, burnZone, ""))
 			{
 				return true;
 			}
+			else
+			{
+				set_property("auto_nextEncounter","");
+			}
 		}
 		if(wannaDigitize)
 		{
 			auto_log_info("Burn some delay somewhere (digitize), if we found a place!", "green");
+			set_property("auto_nextEncounter",get_property("_sourceTerminalDigitizeMonster"));
 			if(autoAdv(burnZone))
 			{
 				return true;
 			}
+			else
+			{
+				set_property("auto_nextEncounter","");
+			}
 		}
 		if(wannaSausage)
```

Comment:
Is there a reason for not setting auto_nextEncounter when sausage goblins are up?

Reply:
it is set in the sausage function, once it has determined to make a autoAdv request

Reply:
Adding a comment to explain this would be nice